src/handlers/comments/getComments.js

```diff
@@ -39,9 +39,10 @@ function atomFeedForItems(items, target) {
 
     // Excerpt regex taken from https://stackoverflow.com/a/5454297
     const excerpt = (str) => str.replace(/\s+/g, ' ').replace(/^(.{40}[^\s]*).*/, '$1');
+    const cleanHtmlTags = (str) => str.replaceAll(/<[\/]?[A-Za-z0-9]*[<A-Za-z0-9>]?/g,'')
```

Comment:
We already have a `stripTags()` function. It would be better to move that to https://github.com/datenanfragen/backend/blob/master/src/util/functions.js and reuse it in both places:

https://github.com/datenanfragen/backend/blob/55745c5a7d72f5845360b4e87bca97fe5389980e/src/handlers/comments/putComment.js#L68-L70

Reply:
👍🏻

Reply:
Just note that this doesn't really cover open tags case:

```
str = "bla bla <link> asdasd asd a </link> asdas <link"
'bla bla <link> asdasd asd a </link> asdas <link'
str.replace(/<[^>]+>/gi, '')
'bla bla  asdasd asd a  asdas <link'
```

Reply:
Modified it to:
`str.replace(/<[^>]+[>]?/gi, '')`

Reply:
It does work if you call `stripTags()` on the text itself rather than the excerpted version. :) I would rather not remove unclosed tags, especially not in general (here, it doesn't matter too much but the other use would actually change what we store in the database).

Reply:
As I said, this won't be a problem if you do `excerpt(stripTags(item.message))` instead of `stripTags(excerpt(item.message))`, as `item.message` contains a full tag in this case.

Reply:
👍🏻